hnn_core/network.py

```diff
@@ -410,9 +412,10 @@ def _create_all_spike_sources(self):
                             params=self.p_common[p_ind],
                             gid=gid))
 
-                # create the cell and artificial NetCon
-                pc.cell(gid, self.common_feeds[-1].connect_to_target(
-                        self.params['threshold']))
+                event_times = self.common_feeds[-1].event_times
```

Comment:
why do we still append the feeds to a list? it's not needed anymore

Reply:
You're correct, my bad.